app/scripts/modules/core/src/help/HelpMenu.tsx

```diff
@@ -17,6 +17,22 @@ const Feedback = () =>
     </MenuItem>
   ) : null;
 
+const AdditionalHelpLinks = () =>
+  SETTINGS.additionalHelpLinks && SETTINGS.additionalHelpLinks.length ? (
+    <>
+      {SETTINGS.additionalHelpLinks.map((helpLink, i) => (
+        <MenuItem href={helpLink.url} key={i} target="_blank">
+          {helpLink.icon ? (
+            <span>
+              <i className={helpLink.icon} /> &nbsp;
+            </span>
+          ) : null}
+          {helpLink.text || `Additional Help ${i}`}
```

Comment:
```suggestion
          {helpLink.text || `Additional Help`}
```
I would drop the `${i}` part here. My guess is, if folks omit the `text` part of the help, they're only adding a single additional help link, so it would look weird to see `Additional Help 0`.